public/lessons/client_side_coding/jquery.md

```diff
@@ -180,10 +180,12 @@ In the [prototype JS framework](http://www.prototypejs.org) the function `bind`
 
 This is useful for passing callbacks that still refer to their own objects, rather than whatever random thing `this` is set to at the time they're called.
 
-Maybe jQuery's `bind` method should have been called `listen` or `handle`, since it's event specific. Oh well.
+Maybe jQuery's `bind` method should have been called `listen` or `handle`, since it's event-specific. Oh well.
 
 jQuery's version of prototypejs's `bind` method is called `proxy`, which is kind of a poor name since it doesn't really implement the Proxy Pattern, or help you to do so.
 
+The [fat arrow](https://www.sitepoint.com/es6-arrow-functions-new-fat-concise-syntax-javascript/) `=>` is used in CoffeeScript, ES6, and TypeScript to accomplish this, so `bind` is probably obsolete
```

Comment:
`bind` is still used for higher order functions as values. Maybe we can make a lesson about this.

For Object-Oriented Javascript `=>`  should be used for functions bound to the object that modify properties of the enclosing object. If however an object uses a partially applied function defined in another execution context, then the function will need to be rebound using `bind`.

This (no pun intended) is a big subject and probably deserves a number of lessons dedicated throughout the course of the curriculum.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/bind